backend/utils.py

```diff
@@ -83,6 +86,23 @@ def set_nested_in_dict(d: Dict, key_path: List, value: Any):
     get_nested_from_dict(d, key_path[:-1])[key_path[-1]] = value
 
 
+def return_err_with_trace(func):
```

Comment:
#### `return_err_with_trace` decorator
@Sigfried This is what I wrote to do the universal exception handling on the backend.

Here's what this looks like.
<img width="614" alt="Screen Shot 2023-03-03 at 6 55 59 PM" src="https://user-images.githubusercontent.com/13045020/222858079-76e1a419-1005-45dd-8c9a-c1e25007e27e.png">

The two console errors you see there are expected/intended, but we can talk more about them when I get back.

You can see I've included fields `status`, `error`, and `stacktrace`. I think we should use these to display a proper error notification to the user.